eland/ml/imported_ml_model.py

```diff
@@ -59,14 +63,28 @@ class ImportedMLModel(MLModel):
         - sklearn.tree.DecisionTreeRegressor
         - sklearn.ensemble.RandomForestRegressor
         - sklearn.ensemble.RandomForestClassifier
+        - lightgbm.LGBMRegressor
+            - Categorical fields are expected to already be processed
+            - Only the following objectives are supported
+                - "regression"
+                - "regression_l1"
+                - "huber"
+                - "fair"
+                - "quantile"
+                - "mape"
+            - The following booster types are supported
```

Comment:
I'm not sure I'd bother to mention booster types here. This is a training detail and so doesn't affect inference.

Reply:
I suppose, but we do check the booster type to make sure it is a model type that we actually support. Example, XGBoost supports a linear regression booster that results in a completely different format.

Reply:
Ok gotcha. I see now see the source of this. This is unfortunate because different concepts are conflated as booster between the two libraries. What xgboost labels booster is really the base learner as opposed to what GBM calls booster, which does actually select the boosting algorithm. In that case, I'd be inclined to somehow disambiguate the concepts in the converters.